codalab/apps/web/templates/account/signup.html

```diff
@@ -38,6 +38,13 @@
                 <input id="id_confirmation_key" name="confirmation_key" type="hidden">
             </div>
 
+            {% if settings.MAILCHIMP_API_URL %}
```

Comment:
purrrfect, can have an `{% else %}` with...

```html
<input type='hidden' id="id_newsletter_opt_in" value="">
```

Or what-have-you, above may not work exactly correctly but I believe empty value should evaluate to false

Reply:
Worked great! Thanks!